packages/synapse-react-client/src/lib/containers/entity_finder/EntityFinderModal.tsx

```diff
@@ -39,39 +38,35 @@ export const EntityFinderModal = (props: EntityFinderModalProps) => {
           setShowUnsavedChangesWarning(false)
         }}
       />
-      <FluidModal
-        className={showUnsavedChangesWarning ? 'SRC-hidden' : undefined}
-        show={props.show}
+      <ConfirmationDialog
+        open={props.show}
         title={props.title}
-        titlePopoverProps={props.titlePopoverProps}
-        onClose={props.onClose}
-        primaryAction={{
-          copy: props.confirmButtonCopy,
-          onClick: () => {
-            props.onConfirm(selected)
-          },
+        fullWidth={false}
+        maxWidth="lg"
+        titleHelpPopoverProps={props.titleHelpPopoverProps}
+        confirmButtonText={props.confirmButtonCopy}
+        onConfirm={() => {
+          props.onConfirm(selected)
+        }}
+        onCancel={() => {
+          if (selected.length > 0) {
+            setShowUnsavedChangesWarning(true)
+          } else {
+            props.onCancel()
+          }
         }}
-        secondaryActions={[
-          {
-            copy: 'Cancel',
-            onClick: () => {
-              if (selected.length > 0) {
-                setShowUnsavedChangesWarning(true)
-              } else {
-                props.onCancel()
-              }
-            },
-          },
-        ]}
-      >
-        <>
-          <Typography variant={'body1'}>{props.promptCopy}</Typography>
-          <EntityFinder
-            {...props.configuration}
-            onSelectedChange={setSelected}
-          />
-        </>
-      </FluidModal>
+        content={
+          <Box width="min-content">
```

Comment:
Prevent text from expanding the dialog to the maxWidth when not in search mode in SinglePane